File: RSC-2019/Round-1/prime number/Solution.cpp
#include <iostream>

int prime(int n) {
    int c, i;
    c = 0;
    for (i = 2; i < n; i++) {
        if (n%i == 0) {
            c++;
        }
    }
    return !c;
}

int main() {
    int n;
    std::cin >> n;
    if (n <= 1) {
        std::cout << "NONE";
    } else {
        if(prime(n)) {
            std::cout << "PRIME";
        } else {
            std::cout << "NOT PRIME";
        }
    }
}

File: RSC-2019/Round-1/prime number/Solution.py
def prime(n: int):
    c = 0
    for i in range(2, n):
        if n%i == 0:
            c+=1
    return c == 0

n = int(input())
if n <= 1:
    print("NONE")
else:
    print("PRIME") if prime(n) else print("NOT PRIME")